pages/story/story_data.js

```diff
@@ -0,0 +1,28 @@
+
+//write functions here to take data and push to story_array
+
+export const story_array = 
+[{
+    num : 1,
+    href: "/story/1", //keep this unique
+    teampic: "https://images.unsplash.com/photo-1628891890467-b79f2c8ba9dc?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Mnx8c3BvcnRzJTIwdGVhbXxlbnwwfHwwfHw%3D",
+    profilepic: "https://images.unsplash.com/photo-1527980965255-d3b416303d12?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8NXx8YXZhdGFyfGVufDB8fDB8fA%3D%3D",
+    name: "Ramkumar V.",
+    date: "2022/23",
+    title: "The one where we beat Goa :)",
+    brief_content: "Here are the biggest achievements, fun and some nerve cracking moments of last year!",
+    body_content:"Hello how is it going"
+},
+{
+    num : 2,
+    href: "/story/2", //keep this unique
+    teampic: "https://images.unsplash.com/photo-1628891890467-b79f2c8ba9dc?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Mnx8c3BvcnRzJTIwdGVhbXxlbnwwfHwwfHw%3D",
+    profilepic: "https://images.unsplash.com/photo-1527980965255-d3b416303d12?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8NXx8YXZhdGFyfGVufDB8fDB8fA%3D%3D",
+    name: "Sudhanshu C.",
+    date: "2022/23",
+    title: "The one where we beat Hyd :)",
+    brief_content: "Here are the biggest achievements, fun and some nerve cracking moments of last year!",
+    body_content:"Its going fine"
+}
+
```

Comment:
empty line. run prettier on all files which will fix all these and tabs and then push a commit